garfield/lookup/views.py

```diff
@@ -0,0 +1,94 @@
+#from celery import chain
+#from celery import shared_task
+
+#from django.conf import settings
+#from django.db.models.signals import post_save
+#from django.dispatch import receiver
+#from django.forms.models import model_to_dict
+#from django.template.loader import render_to_string
+
+import requests
+#import sys
+
+#from django.shortcuts import render
+from django.http import HttpResponse
+
+from twilio.rest import Client
+from twilio.twiml.messaging_response import MessagingResponse
+
+from contacts.models import Contact
+from .decorators import twilio_view
+from garfield import local as local
```

Comment:
Django provides a function for getting all configuration properties - you can do that using:

`from django.conf import settings`

All your configuration options are then available off of the `settings` object, so for Twilio credentials you can do:

`settings.TWILIO_ACCOUNT_SID`
and
`settings.TWILIO_AUTH_TOKEN`